src/app/client/src/app/modules/learn/components/course-consumption/assessment-player/assessment-player.component.ts

```diff
@@ -382,6 +382,9 @@ export class AssessmentPlayerComponent implements OnInit, OnDestroy, ComponentCa
   public getCurrentContent() {
    return this.previousContent ? this.previousContent : this.activeContent;
   }
+  public getActiveContent() {
+    return this.previousContent ? this.previousContent : this.activeContent;
```

Comment:
@rajeevsathish What is the difference between getCurrentContent() & the new method getCurrentContent().

we could have used the same method right(getCurrentContent()) instead of new one.